pkg/skuba/actions/cluster/images/images.go

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2019 SUSE LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package cluster
+
+import (
+	"fmt"
+	"k8s.io/kubernetes/cmd/kubeadm/app/images"
+	"github.com/SUSE/skuba/internal/pkg/skuba/kubernetes"
+	"github.com/SUSE/skuba/pkg/skuba"
+)
+
+// Print out list of images that will be pulled
+// This can be used as input to skopeo for mirroring in air-gapped scenarios
+func Images() error {
```

Comment:
I'm not fully sure if that's compatible with what we do in [`cluster status`](https://github.com/SUSE/skuba/blob/master/pkg/skuba/actions/cluster/status/status.go#L47-L56) but adding column titles would make it look better

also I would put all image names into an array/map, so we can loop through them and don't repeat the `fmt.Println` so much in the code

Reply:
I have updated the output to match that of the status command and now iterate over addons and components.